ios/NotifeeCore/NotifeeCore+UNUserNotificationCenter.m

```diff
@@ -158,7 +158,11 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
   NSDictionary *notifeeNotification =
       response.notification.request.content.userInfo[kNotifeeUserInfoNotification];
 
-  // we only care about notifications created through notifee
+  // handle notification outside of notifee
+  if (notifeeNotification == nil) {
+    notifeeNotification = [NotifeeCoreUtil parseUNNotificationRequest:response.notification.request];
```

Comment:
FYI it looks like there is a conflict with the Braze SDK with this line
To reproduce : install notifee and braze. Open your app, send a Braze test IAM campaign, click on the notification to show the IAM. No IAM is shown.
And possibly conflicts with other SDKs

Reply:
Workaround is patching parseUNNotificationRequest with 

```patch
+  if (userInfo != nil && userInfo[keyToExclude] != nil) {
+      return nil;
+  }
```

Reply:
Oh no! Looks like an easy enough local patch-package fix for you (I'd bet $1 you already have that running ;-) ) but a PR here is something we could integrate